src/CodeGenerator/src/Generator/RequestSerializer/RestXmlSerializer.php

```diff
@@ -105,6 +105,7 @@ private function dumpXmlShape(Member $member, Shape $shape, string $output, stri
         switch ($shape->getType()) {
             case 'blob':
             case 'string':
+            case 'integer':
```

Comment:
This is the only change I made in the generator. 

Im not 100% convinced that it is correct. I'll check that later.

Reply:
should work. Except if people provide non-base10 integer 

Maybe you could add test that include one of those
```
$a = 0123;
$a = 0x1A;
$a = 0b11111111;
```

Reply:
Those should be handled the same in the XML.